src/world_interface/real_world_constants.h

```diff
@@ -83,8 +83,8 @@ constexpr auto potMotors = frozen::make_unordered_map<motorid_t, potparams_t>(
 	  {.adc_lo = 123, .mdeg_lo = -200 * 1000, .adc_hi = 456, .mdeg_hi = 200 * 1000}},
 	 {motorid_t::forearm,
 	  {.adc_lo = 1208, .mdeg_lo = -180 * 1000, .adc_hi = 841, .mdeg_hi = 180 * 1000}},
-	 {motorid_t::wrist,
-	  {.adc_lo = 123, .mdeg_lo = -100 * 1000, .adc_hi = 456, .mdeg_hi = 100 * 1000}},
+	 //{motorid_t::wrist,
+	 //{.adc_lo = 123, .mdeg_lo = -100 * 1000, .adc_hi = 456, .mdeg_hi = 100 * 1000}},
```

Comment:
You can remove this line, and can you add entries for the wristDiffLeft and wristDiffRight? Just dummy values in the map is fine (all 0s) we'll have to calibrate it when it's mounted